vizro-ai/docs/pages/user-guides/genAI-use-case.md

```diff
@@ -0,0 +1,105 @@
+# How to add Vizro-AI created charts into a Vizro dashboard
+
+This guide provides instructions on incorporating a chart generated by Vizro-AI into your [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
+
+1. Create a chart with Vizro-AI that you would like to visualize in a dashboard.
+
+    In this example, we aim to create a chart that illustrates the population development of each continent over time. To gain deeper insights and access the underlying code responsible for generating the chart, include `explain=True` in the `plot()` method. Let's execute the provided code and examine the outcome.
+
+    !!! example "Vizro-AI chart"
+        === "Code for the cell"
+            ```py
+            import vizro_ai
+            from vizro_ai import VizroAI
+            import plotly.express as px
+            from dotenv import load_dotenv
+
+            load_dotenv()
+
+            df = px.data.gapminder()
+            vizro_ai = VizroAI(model_name="gpt-4-0613")
+
+            vizro_ai.plot(df,
+                          """Plot a bubble chart to show the changes in life expectancy
+                           and GDP per capita for each country over time.
+                           Color the bubbles by continent.
+                           Add animation on yearly basis, and do not use facets.
+                           Put the legend on top""")
+            ```
+        === "Result"
+            [![VizroAIChart]][VizroAIChart]
+
+        [VizroAIChart]: ../../assets/user_guides/vizro-ai-chart.gif
+
+2. Insert the resulting chart into a dashboard.
+
+    Once you are satisfied with the chart, you incorporate the chart into a [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard. In the explanation section from the result of the example above, you'll find the code snippet used to generate the chart.
```

Comment:
Is there an explanation section if you don't pass `explain=True` to the `plot()` method?

Reply:
```suggestion
    Once you are satisfied with the chart, you can add it to a [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard. In the explanation section from the result of the example above, you'll find the code snippet used to generate the chart.
```